reflekt-plugin/src/main/kotlin/org/jetbrains/reflekt/plugin/scripting/ImportChecker.kt

```diff
@@ -50,6 +46,7 @@ class ImportChecker {
                 } catch (e: Throwable) {
```

Comment:
What kind of exceptions you tried to catch here? This is not a good practice to catch Throwable

Reply:
We use Reflection at the compile-time with the `compileClasspath` configuration imports and we can catch some unexpected errors like `SecurityException` or `NoClassDefFoundError` since some classes can not be loaded at the compile time for an unexpected reason. 
This can happen if, for example, the included libraries use other libraries that may not be available at compile time.
I know that it is bad practice to catch too general errors, but I am not sure that we can predict the java reflections behaviour here. 
Specifically, in this case, it seems to me that we can leave this method of error handling.  At least until we find a really good way to test this behaviour, for example, fuzzing would be nice here I think.

In any case, filtering is only used in the Smart Reflect part and does not affect the main functionality of the plugin. What do you think?

Reply:
Yes, it makes sense, I totally forgot about things like `NoClassDefFoundError`. Let's suppress it then

Reply:
maybe we should add a comment in the code to clarify why we are using such a common error here

Reply:
done